content/blog/get-started-with-application-performance-monitoring-tools-overview.md

```diff
@@ -0,0 +1,233 @@
+
+Dynatrace offers minimal alerting, but almost no problem/incident management features out-of-the-box. The third-party incident management and status page solutions must be integrated. Dynatrace has no capabilities in the area of federated, hierarchical, or edge _AI/ML_. 
+
+Dynatrace offers a full-stack pricing model, starting at _$74/month_ per 8 GB per host. It also offers individual product pricing models, such as infrastructure monitoring, digital experience monitoring, application security and open ingestion, etc. Each of those pricing models works as an add-on and is not included in the full-stack. They are charged with additional cost.
+
+### **Elastic**
+
+![](/img/elastic.png)
+
+[Elastic](https://www.elastic.co/) is a distributed search and analytics solution. 
+
+Key features:
+
+* operate in a distributed environment with scalability and resiliency;
+* allow full control over data, users and cluster operations with a variety of management tools, such as snapshots, index lifecycle, data tiers, data streams;
+* protect data with a list of security features, such as _keystore_,  encrypted communications, RBAC, IP filtering, security realms, SSO and audit logging;
+* support customized and reliable alerting and notification integration with any other third-party systems;
+* allow to work with data using various language clients, Elasticsearch _DSL_ and _SQL_, and _REST APIs_;
+* extend Elasticsearch functionality with various plugins and integrations;
+* run and manage Elasticsearch across public cloud, private cloud and Kubernetes using _Elastic Cloud_, _Elastic Cloud Enterprise_ and _Elastic Cloud on Kubernetes_;
+* ingest any data type using language clients, ingest nodes, lightweight shippers or _Logstash_;
+* enrich raw data using a variety of analyzers, tokenizer, filters, and enrichment options;
+* support document storage, time series analysis and metrics, and geospatial analytics;
+* provide full-text search capabilities with its inverted index, tunable relevance scoring and advanced query _DSL_;
+* find data relationships through aggregations and graph exploration and create alerts;
+* model and automate the analysis of time series data, combine alerting and inference using machine learning;
+
+Elastic builds and maintains the *Elastic Stack*, an _all-in-one_ platform built upon the proven *ELK Stack* for the logs, metrics, and application trace data with a multitude of out-of-the-box integrations. Elastic Stack is the foundation for its primary solutions, *Elastic Enterprise Search*, the fleet of search solutions, *Elastic Observability*, the solution for unified visibility across logs, metrics and APM data, and *Elastic Security*, the solution that unifies endpoint protection and _SIEM_. You can easily deploy any of these solutions as a managed service with Elastic Cloud, with one stack powering three solutions. 
```

Comment:
...upon the proven *Elasticsearch, Logstash, and Kibana (ELK) Stack* for the logs, ...